### PR TITLE
Add Travis and Setup Deploy Pipelin

### PR DESCRIPTION
**Pull request by boldtrn**
With this PR, every commit into master will be deployed to Sonatype. The version can be set in the file `gradle.properties`.

Currently, releases are unsigned.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,4 +13,19 @@ android:
...
+- make build-release
+
+after_success:
+  # deploy snapshot artifacts to sonatype
+  - if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
+        echo "release to maven central";
+
+        # Echo username and password into gradle.properties
+        # This will be picked up in gradle/mvn-push-android.gradle
+        echo "NEXUS_USERNAME=$CI_DEPLOY_USERNAME
```
Comment (karussell) [inline .travis.yml]: Oh. Please don't do that, as logs of OS are visible for everyone even later on. Will have to change them ...

Comment (karussell) [reply .travis.yml]: Ah, could it be that the newline is a mistake?

Comment (boldtrn) [reply .travis.yml]: > Ah, could it be that the newline is a mistake?

I think the newline should work, but we could split it into two separate echos?

You can try it in your local console, echo understands the newline and will put the block
```
NEXUS_USERNAME=$CI_DEPLOY_USERNAME
NEXUS_PASSWORD=$CI_DEPLOY_PASSWORD
```

Into the file gradle.properties. I think this should not appear in the system logs.

But I think you are right, it's probably better if we read these values from the environment variables. I will update the PR.

**Comment by boldtrn** [comment]
I updated the PR. I am not 100% sure that it will work that way, but it might be worthwhile trying?

**Pull request merged**